File: day 018/classwork/classwork02.py
# 2) შექმენით ისეთ პროგრამა რომელიც მომხმარებელს შემოატანინებს 2 რიცხვს.
# შემოტანის შემდეგ for loop ის მეშვეობით გამოთვალეთ ყველა რიცხვი ამ ორ
#  რიცხვს შორის და შემდეგ ისიდი ჩაამატეთ ლისტში საბოოლოოდ კი დაპრინტეთ ტერმინალში
numbers = []
num1 = int(input("enter number: "))
num2 = int(input("enter number: "))
if num1 > num2:
    for i in range(num2 + 1, num1):
        numbers.append(i)
else:
    for i in range(num1 + 1, num2):
        numbers.append(i)
print(numbers)

File: day 083/classwork/classwork02.js
let num1 = Number(prompt("Enter first number: "));
let num2 = Number(prompt("Enter second number: "));

let operator = prompt("Choose operator(+, -, *, /): ");
let res = "";

if (operator === "+") {
    res = num1 + num2;
} else if (operator === "-") {
    res = num1 - num2;
} else if (operator === "*") {
    res = num1 * num2;
} else if (operator === "/") {
    if (num2 === 0) {
        alert("ნული არ შეიძლება იყოს გამყოფი");
        res = ""; 
    } 
    else if (num1 === 0) {
        alert("ნული არ შეიძლება იყოს გამყოფი");
        res = "";   
    }
    else {
        res = num1 / num2;
    }
} else {
    alert("რაღაცა უცხო გაქვს შეყვანილი");
    res = ""; 
}

if (res !== "") {
    alert(res);  
}